tasks/affective_task/server_affective_task.py

```diff
@@ -88,7 +92,7 @@ def run(self, images_dir: str, collaboration: bool = False):
                             "selected_rating_type": response["update"]["rating_type"],
                             "selected_rating": response["update"]["rating_index"] - 2
                         }
-                        self._csv_writer.writerow([time(), image_path, client_name, json.dumps(record_activity)])
+                        self._csv_writer.writerow([time(), monotonic_time, boot_time, image_path, client_name, json.dumps(record_activity)])
```

Comment:
Does not look like we need `monotonic_time` and `boot_time` anywhere else than inside the `writerow` method. Furthermore, the monotonic time and boot time recorded before the `receive` function are delayed because the `receive` method freezes the program until it receives data from the client.

do this instead
```
self._csv_writer.writerow([time(), monotonic(), psutil.boot_time(), image_path, client_name, json.dumps(record_activity)])
```